assets/docs/vasudevgupta7/models/wav2vec2-base/1.md

```diff
@@ -0,0 +1,50 @@
+# Module vasudevgupta7/wav2vec2-base/1
+
+Pre-trained speech model (without any head) from Facebook for Automatic Speech Recognition
+
+<!-- asset-path:  https://huggingface.co/vasudevgupta/tf-wav2vec2-base/resolve/main/wav2vec2-base.tar.gz -->
+<!-- task: audio-stt -->
+<!-- network-architecture: wav2vec2-base -->
+<!-- format: saved_model_2 -->
+<!-- fine-tunable: true -->
+<!-- license: apache-2.0 -->
+<!-- language: en -->
+<!-- colab: https://colab.research.google.com/github/vasudevgupta7/gsoc-wav2vec2/blob/main/notebooks/wav2vec2_base_saved_model.ipynb -->
+
+## Overview
+
+This model is TensorFlow equivalent of PyTorch [`facebook/wav2vec2-base`](https://huggingface.co/facebook/wav2vec2-base). This model was introduced in the paper: [wav2vec 2.0: A Framework for Self-Supervised Learning of Speech Representations](https://arxiv.org/abs/2006.11477).
+
+**How to use this model?**
+
+Add randomly initalized LM head over the top of pre-trained model & fine-tune the whole model.
+
+```python
+import tensorflow as tf
+import tensorflow_hub as hub
+
+loaded = hub.load("https://tfhub.dev/vasudevgupta7/wav2vec2-base/1")
+
+# For using this pre-trained model for training, just pass `trainable=True` in `hub.KerasLayer`
+pretrained_layer = hub.KerasLayer(loaded.signatures["wav2vec2"], trainable=True)
```

Comment:
Since there is only one signature, does it work to call `hub.KerasLayer("https:/....base/1")` directly? This provides the most seamless experience for users.